docker/docker_client_test.go

```diff
@@ -87,7 +87,178 @@ func TestDockerCertDir(t *testing.T) {
 	}
 }
 
-func TestGetAuth(t *testing.T) {
+func TestGetAuthNewPath(t *testing.T) {
```

Comment:
Is this really that different from `TestGetAuthOldPath`? Could this just be a nested loop, running the same set of tests twice, once for each path? Or a loop calling a helper with the other loop, or something like that?

Reply:
fixed.